docs/topics/rn-resolved-issues.adoc

```diff
@@ -7,7 +7,7 @@
 
 At the time of the release the following resolved issues have been identified as major issues worth highlighting.
 
-For a complete list of all issues resolved in this release, see the list of link:https://issues.redhat.com/issues/?filter=12364384[{ProductVersion} resolved issues] in Jira.
+For a complete list of all issues resolved in this release, see the list of link:https://issues.redhat.com/browse/WINDUP-3125?filter=12385755[{ProductVersion} resolved issues] in Jira.
```

Comment:
The {ProductVersion} attribute needs to be updated to 5.2.1 because that is what the Jira search resolves to.

But for this release, you should comment out this line. There is no need for a "complete list of all issues" (repetition!) because there are only 3 issues and you include them in the table.

Reply:
The {ProductVersion} will be updated in a separate PR as mentioned in the PR's description.